tests/src/JIT/config/benchmark/project.json

```diff
@@ -14,6 +14,10 @@
     "System.Threading": "4.0.10",
     "System.Threading.Tasks": "4.0.10",
     "System.Threading.Tasks.Parallel": "4.0.0",
+    "System.Diagnostics.Process": "4.0.0-*",
```

Comment:
These probably shouldn't use star versioning.  You'll get shifting behavior over time as package versions change.  Either that or check in the project.lock.json.